client-src/elements/chromedash-activity-log.ts

```diff
@@ -77,11 +77,13 @@ export class ChromedashAmendment extends LitElement {
 customElements.define('chromedash-amendment', ChromedashAmendment);
 
 export interface User {
+  //will move to chromedash-app.js after migration
```

Comment:
It might help to formalize this more as a TODO.

```suggestion
  // TODO(markxiong0122): Move this interface to chromedash-app.js after migration.
```